apps/back-office/src/app/api/diagnosis-treatment-plan/route.ts

```
@@ -181,13 +216,73 @@ export async function PUT(request: NextRequest) {
+    // First, get the existing plan to check if it has a survey answer
+    const existingPlan = await prisma.diagnosisTreatmentPlan.findUnique({
+      where: { id },
+      include: { SurveyAnswers: true },
+    });
+
     const result = await prisma.$transaction(async (tx) => {
+      let finalSurveyAnswerId: string | null =
+        existingPlan?.survey_answers_id || null;
+
+      // Handle survey answer - update existing or create new
+      if (surveyData && surveyData.templateId && surveyData.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)